docs/source/contributing.rst

```diff
@@ -33,6 +33,75 @@ Furthermore we value a systematic approach to development with requirements
 analysis and evaluation of design options as well as iterative design with fast
 test and review cycles.
 
+.. _authorship:
+
+Authorship policy
+-----------------
+
+Since the performance of scientists is evaluated based on the number and quality
+of their publications, authorship is an important topic for software with a
+focus on scientific application, such as LiberTEM. Our goal is to give
+appropriate credit to each contribution so that the work of contributors becomes
+visible as a form of scientific output and is considered in the evaluation of
+their productivity as scientists. We hope that this can support and encourage
+Open Source software development as a form of scientific work.
+
+Following established practices for scientific papers, we distinguish between
+creators, who are persons that contributed in a way that establishes
+co-authorship of LiberTEM, and contributors, who are persons that contributed in
+other ways, for example through discussions or help.
+
+Creators are persons who
+
+* Contributed at least one commit to the repository.
+* Contributed material such as code or documentation in other ways that don't
+  appear directly as commit, including prototype code that served as a basis for
+  code in LiberTEM.
+* Contributed to the management and direction, for example
+  active management and support for contributors who dedicate a significant part
+  of their working time to LiberTEM.
```

Comment:
This statement still looks not very well defined to me, but since there is no better idea, it can be let as it.